api/server/router/image/image_routes.go

```diff
@@ -337,6 +337,22 @@ func (ir *imageRouter) getImagesJSON(ctx context.Context, w http.ResponseWriter,
 		return err
 	}
 
+	for _, img := range images {
+		if versions.LessThan(version, "1.43") {
+			if len(img.RepoTags) == 0 && len(img.RepoDigests) == 0 {
+				img.RepoTags = append(img.RepoTags, "<none>:<none>")
+				img.RepoDigests = append(img.RepoDigests, "<none>@<none>")
```

Comment:
We should probably have a look at `docker/cli` and have code there to keep the presentation the same. ISTR we did something for this in other parts of the code; we should probably already be able to apply that change (if empty, make it `<none>:<none>` when presenting) and even can consider including it in the current version of the CLI if the change is small.